File: 프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.js
function solution(a, b) {
    var answer = 0;
    if(a%2!==0 && b%2!==0){
       answer = a*a+b*b;
    }else if(a%2!==0 || b%2!==0){
        answer = (a+b)*2;
    }else{
        answer = Math.abs(a - b);
    }
    return answer;
}

File: 프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.py
def solution(a, b):
    sumNum = a + b
    if sumNum % 2 != 0:
        return 2 * sumNum
    else:
        if a % 2 == 0 and b % 2 == 0:
            return abs(a - b)
        else:
            return a ** 2 + b ** 2